tests/unit/confidant/encrypted_settings_test.py

```diff
@@ -0,0 +1,48 @@
+import unittest
+from mock import patch
+from mock import Mock
+
+# Prevent call to KMS when app is imported
+from confidant import settings
+settings.encrypted_settings.secret_string = {}
+settings.encrypted_settings.decrypted_secrets = {'SESSION_SECRET': 'TEST_KEY'}
```

Comment:
Can you do this as part of a setUp/tearDown sequence? We have some other tests to use as examples. I worry that this mocking will affect other tests, depending on the run order.